debian/gregoriotex.install

```diff
@@ -1,6 +1,6 @@
-tex/gregorio*.tex /usr/share/texmf/tex/luatex/gregoriotex
-tex/gsp-default.tex /usr/share/texmf/tex/luatex/gregoriotex
-tex/gregorio*.lua /usr/share/texmf/tex/luatex/gregoriotex
-tex/gregorio*.sty /usr/share/texmf/tex/luatex/gregoriotex
```

Comment:
Thank you for the reminder!  I have now updated the pull request to use `install-gtex.sh` instead.